document-store/src/main/java/org/hypertrace/core/documentstore/expression/operators/FunctionOperator.java

```diff
@@ -11,4 +11,5 @@ public enum FunctionOperator {
   DIVIDE, // Dividend and then divisors (Left to right associative),
   MULTIPLY,
   SUBTRACT, // Minuend and then subtrahends (Left to right associative),
+  TO_LOWER_CASE
```

Comment:
Nit: Leave room for more enums to keep the git history clean.
```suggestion
  TO_LOWER_CASE,
```

Reply:
This will go under the "Unary operators" section above, right?

Reply:
I don't understand the reason for `,` can you explain it

Reply:
Nice article explaining it: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8